public_suffix_list.dat

```diff
@@ -13760,4 +13760,11 @@ basicserver.io
 virtualserver.io
 enterprisecloud.nu
 
+// Diher Solutions : https://diher.solutions
```

Comment:
Need to move the added section up to a higher place in the PRIVATE DOMAINS section, sorted ascending alphabetically by your org name in order to comply with guidelines

Reply:
fixed